Jenkinsfile

```diff
@@ -121,7 +121,8 @@ el7_component_repos = ""
 def functional_rpms  = "--exclude openmpi openmpi3 hwloc ndctl " +
                        "ior-hpc-cart-4-daos-0 mpich-autoload-cart-4-daos-0 " +
                        "romio-tests-cart-4-daos-0 hdf5-tests-cart-4-daos-0 " +
-                       "mpi4py-tests-cart-4-daos-0 testmpio-cart-4-daos-0 fio"
+                       "mpi4py-tests-cart-4-daos-0 testmpio-cart-4-daos-0 " +
+                       "fio MACSio"
```

Comment:
Pity I didn't see this PR.  Adding test tool RPMs to `functional_rpms` should be deprecated and they should be added as dependencies of the _daos-tests_ RPM.  I just haven't gotten to making that change yet unfortunately to revert the existing ones.